app/helpers/application_helper.rb

```diff
@@ -22,11 +22,11 @@ def encoded_home_url
   end
 
   def social_image
-    image_url('social-nairobi-national-park.jpg')
+    image_url('social-le-thi-luan.jpg')
   end
 
   def social_image_alt
-    'Rhino, Nairobi skyline, Nairobi National Park, Kenya'
+    'Le Thi Luan tends to her rice paddy, Vietnam'
```

Comment:
hahaha, I can only wish...

This has actually alerted me to something. Seems like the `social_image` and `social_image_alt` helper methods are still using the old hero image. This should be changed to the new 'windturbines on the ocean' hero image we have, but I don't know what we should use as the title/alt text. Will come back to this

Reply:
> > Checking through on the site and it all looks top! I only spotted two small things while testing in the browser on chrome.
> > ```
> > * [x]  It's slightly odd how the share icon remains but does sit flush to the gutter on the right (because the burger icon is still there, just invisible). Maybe check with Charlotte on this one to see if it needs changing. Can be done separately to this PR though.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ![Screenshot 2021-06-01 at 13 48 18](https://user-images.githubusercontent.com/43963744/120325972-1c686a80-c2e0-11eb-9567-2f149bc0feba.png)
> 
> On this point, I've added in a `flex-row-reverse` class on the flex container. It feels a little bit clumsy but it seems to have come out with more or less the right effect (see screenshot below). There is still a wrapper div with a `gutter-right` class that pushes the share button out from the side a little but, but can address that later. I'll get in touch with Charlotte as you suggested to see what she thinks
> 
> <img alt="image" width="1695" src="https://user-images.githubusercontent.com/56026099/120816857-10331600-c549-11eb-9749-8521750aeda2.png">

👍  Just to check, does it switch back when transitioning to the full nav on scroll?

Reply:
Ah shucks, this has actually broken it for the full scrolling navbar. Nice catch.

<img width="942" alt="image" src="https://user-images.githubusercontent.com/56026099/120993115-26281d00-c77b-11eb-8647-36e6bc13f803.png">

I'll look into whether this is easily fixable...hopefully so